src/Playwright/Core/PlaywrightImpl.cs

```diff
@@ -76,6 +76,8 @@ internal PlaywrightImpl(IChannelOwner parent, string guid, PlaywrightInitializer
 
         public ISelectors Selectors { get => _initializer.Selectors; }
 
+        public PlaywrightInitializer Intitializer { get => _initializer; }
```

Comment:
Why are we exposing this?

Reply:
Aah yup initializer is not required. PreLaunchedBrowser is required which can be retrieved by `_initializer.PreLaunchedBrowser`

Reply:
fixed